api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -75,7 +76,7 @@
         request.getGivenName(), request.getFamilyName(), request.getUsername(),
         request.getPassword()
     );
-    return null;
+    return ResponseEntity.status(HttpStatus.CREATED).build();
```

Comment:
Have you checked to make sure that the generated TypeScript can actually handle ResponseEntity<Void> in this manner?

(I guess if it can't, you can change this later...)